src/schema/v2/Alerts/index.ts

```diff
@@ -484,3 +484,32 @@ export const AlertsSummaryFields = {
     resolve: (resp) => resp,
   },
 }
+
+export const PartnerAlertsFields = {
+  id: { type: GraphQLString },
+  searchCriteriaId: {
+    type: GraphQLString,
+    resolve: ({ search_criteria_id }) => search_criteria_id,
+  },
+  partnerId: {
+    type: GraphQLString,
+    resolve: ({ partner_id }) => partner_id,
+  },
+  score: { type: GraphQLString },
+  matchedAt: {
+    type: GraphQLString,
+    resolve: ({ matched_at }) => matched_at,
+  },
+  alert: {
```

Comment:
This object might be better off called more explicitly `PartnerAlertsEdgeFields`, and then this property might be left off (vs. using the more canonical `node`).

Reply:
Excellent idea, addressed [here](https://github.com/artsy/metaphysics/pull/5826/commits/cbfb612b7181d0ec46cffd66d656c677b86fae96)